AntiCovid/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -90,7 +90,11 @@
     <fragment
         android:id="@+id/profileFragment"
         android:name="com.swtug.anticovid.view.profile.ProfileFragment"
-        android:label="ProfileFragment" />
+        android:label="ProfileFragment" >
+        <action
+            android:id="@+id/action_profileFragment_to_loginFragment"
+            app:destination="@id/loginFragment" />
+    </fragment>
```

Comment:
Actually i didnt tested it but what if you log out and then press the back button. I think this will result in turning back to profileFragment. If thats the case please add the two properties @naggab know what i mean to this action or better clear whole backstack when entering login fragment. To dont have any navigation issues.

Since on a new login a complete new flow should start

Reply:
We will fix that.